src/Movies/Movies.js

```diff
@@ -23,5 +24,8 @@ export default function Movies({movies, displayMovie, currentMovieIndex}) {
     )
 }
 
-
-
+Movies.propTypes = {
+    movies: PropTypes.array.isRequired , 
```

Comment:
When checking types of arrays and objects, deep equality is needed to check the type within complex data types of arrays and objets